CHANGELOG.md

```diff
@@ -1,9 +1,9 @@
 # Changelog
 
-## v0.2.0 Beta 6 (Frontend Utils)
-### [0.2.0-beta.6](https://github.com/openfga/frontend-utils/compare/v0.2.0-beta.5...v0.2.0-beta.6) (2024-01-29)
+## v0.2.0 Beta 7 (Frontend Utils)
```

Comment:
You seem to have removed the beta 6 changelog :)

Reply:
Yah, caught it just before you did :P